NinchatSDKSwift/Implementations/View/ViewController/NINChatViewController.swift

```diff
@@ -290,6 +290,8 @@ final class NINChatViewController: UIViewController, KeyboardHandler {
             case .confirm:
                 debugger("WebRTC: Grant permission for the video call")
                 self.viewModel.grantVideoCallPermissions { error in
+                    if permissionError(error) { return }
```

Comment:
Instead of silently failing, it is possible to send ninchat.com/rtc/pick-up message answer: false and unsupported: true in payload. This will decline the call, and tell the agent that customers client does not support video.

Reply:
It does not silently fail, it shows an error banner and sends `hangup` message to disconnect the video

https://github.com/somia/ninchat-sdk-ios-swift/blob/2e9c0090acc90c344be574e92897bb247508837e/NinchatSDKSwift/Implementations/View/ViewController/NINChatViewController.swift#L269

Reply:
But it does not call self.viewModel.pickup(answer: false, unsupported: true), right? I guess that would be the call in Swift.

Reply:
Thanks for the comment, it is done